exercises/forth/canonical-data.json

```diff
@@ -17,13 +17,6 @@
           "1 2 3 4 5"
         ],
         "expected": [1, 2, 3, 4, 5]
-      },
-      {
-        "description": "all non-word characters are separators",
-        "input": [
-          "1\u00002\u00133\n4\r5 6\t7"
```

Comment:
Good catch @petertseng, I saw the unicode escape character and didn't double check if these were in the ASCII range.

I see a couple of choices here:
- Rename the PR/commit message (doesn't align with the original intent of the issue I was trying to fix)
- Revert this part of the change
- Revert this part of the change and replace the unicode representation of the characters with their [hex equivalent](https://msdn.microsoft.com/en-us/library/x9h8tsay.aspx)

I'm in favor of option 3, what do you think?

Also I had a look at the [description.md](https://github.com/robkeim/x-common/blob/6611be0f5e7a73c4d5ec3e5abfa2d92d568700d4/exercises/forth/description.md) and I'll need to update it too:
>You should use the following rules for the syntax: a number is a sequence of one or more (unicode) digits

I can update the PR tonight. What's the policy on making updates to commits? Do I keep the existing history and add a "integrating feedback" commit, or do I re-write the history so that the history looks "clean".